lib/Basic/Default/TaskQueue.inc

```diff
@@ -52,6 +52,18 @@ public:
 } // end namespace sys
 } // end namespace swift
 
+Optional<ResourceStats>
+TaskQueue::OwnResourceStats()
+{
+  return Optional<ResourceStats>();
+}
+
+Optional<ResourceStats>
+TaskQueue::ChildResourceStats()
+{
+  return Optional<ResourceStats>();
+}
```

Comment:
You forgot to update the Finished callback in the default TaskQueue implementation.